src/plugins/shared_ux/public/components/empty_state/no_data_views/documentation_link.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiLink, EuiTitle } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props {
+  documentationUrl: string;
+}
+
+export function DocumentationLink({ documentationUrl }: Props) {
+  return (
+    <dl>
+      <EuiTitle size="xxs">
+        <dt className="eui-displayInline">
+          <FormattedMessage
+            id="sharedUX.noDataViews.learnMore"
+            defaultMessage="Want to learn more?"
+          />
+        </dt>
+      </EuiTitle>
+      &emsp;
+      <dd className="eui-displayInline">
```

Comment:
@cchaos Are we supposed to be using these still, in `emotion`-enabled components?

Reply:
Yes, these are just utility classes and some of them come with more than just a single property that EUI can keep track of.